resolwe_bio/descriptors/unified-descriptor.yml

```diff
@@ -814,6 +814,326 @@
+              value: egfr_e746_a750del
+            - label: EGFR G598V
+              value: egfr_g598v
+            - label: EGFR A289V
+              value: egfr_a289v
+            - label: EGFR
+              value: egfr
+        - name: mutation_status_braf
+          label: Mutation status/BRAF
+          type: basic:string
+          required: false
+          choices:
+            - label: BRAF V640E
+              value: braf_v640e
+            - label: BRAF V640M
+              value: braf_v640m
+            - label: BRAF
+              value: braf
+        - name: mutation_status_alk
+          label: Mutation status/ALK
+          type: basic:string
+          required: false
+          choices:
+            - label: ALK R1275Q
+              value: alk_r1275q
+            - label: ALK F1174L
+              value: alk_f1174l
+            - label: ALK
+              value: alk
+        - name: mutation_status_met
+          label: Mutation status/MET
+          type: basic:string
+          required: false
+          choices:
+            - label: MET
+              value: met
+        - name: mutation_status_ros1
+          label: Mutation status/ROS1
+          type: basic:string
+          required: false
+          choices:
+            - label: ROS1
+              value: ros1
+        - name: mutation_status_cdkn2a
+          label: Mutation status/CDKN2A
+          type: basic:string
+          required: false
+          choices:
+            - label: 'CDKN2A R80*'
+              value: 'cdkn2a_r80*'
+            - label: 'CDKN2A R58*'
+              value: 'cdkn2a_r58*'
+            - label: CDKN2A H83Y
+              value: cdkn2a_h83y
+            - label: CDKN2A
+              value: cdkn2a
+        - name: mutation_status_rbm10
+          label: Mutation status/RBM10
+          type: basic:string
+          required: false
+          choices:
+            - label: 'RBM10 F913='
+              value: 'rbm10_f913='
+            - label: 'RBM10 R745*'
+              value: 'rbm10_r745*'
+            - label: RBM10 S846L
+              value: rbm10_s846L
+            - label: 'RBM10 R97*'
+              value: 'rbm10_r97*'
+            - label: RBM10 Q481L
+              value: 'rbm10_q481l'
+            - label: 'RBM10 R295*'
+              value: 'rbm10_r295*'
+            - label: 'RBM10 E732*'
+              value: 'rbm10_e732*'
+            - label: RBM10
+              value: rbm10
+        - name: mutation_status_ptprd
+          label: Mutation status/PTPRD
+          type: basic:string
+          required: false
+          choices:
+            - label: PTPRD S431L
+              value: ptprd_s431l
+            - label: PTPRD
+              value: ptprd
+        - name: mutation_status_u2af1
+          label: Mutation status/U2AF1
+          type: basic:string
+          required: false
+          choices:
+            - label: U2AF1
+              value: u2af1
+        - name: mutation_status_ntrk3
```

Comment:
Gene NTRK3 has really low expression across many cancers, so it is hard to detect variants using RNA-seq VC in it.